crates/nu-protocol/src/value/mod.rs

```diff
@@ -617,24 +617,29 @@ impl Value {
                     }
                     Value::List { vals, span } => {
                         let mut output = vec![];
+                        let mut hasvalue = false;
+                        let mut temp: Result<Value, ShellError> =
+                            Err(ShellError::IOError("can't follow stream paths".into()));
```

Comment:
I would put here an error that has a span. Otherwise, if you hit this error, it will just print the message but won't show where the error happened. So you can pick an error type that has a span and pass the `orig_span` to it.